src/apps/common/ns_turn_utils.c

```diff
@@ -46,6 +46,14 @@
 
 #include <signal.h>
 
+#include <unistd.h>
```

Comment:
Now need to wrap it with `#if !defined(WINDOWS)` or something similar

Reply:
Done!  I had reorganize the logging part a little, due to changes on main from other pull request merges.